File: app/src/main/java/io/tanjundang/study/knowledge/launchmode/SingleTopActivity.java
package io.tanjundang.study.knowledge.launchmode;

import android.os.Bundle;
import android.view.View;

import io.tanjundang.study.R;
import io.tanjundang.study.base.BaseActivity;

public class SingleTopActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    protected void initView() {
        setContentView(R.layout.activity_single_top);
    }

    @Override
    protected void initData() {

    }

    public void SkipToOne(View v) {
        OneActivity.SkipToActivityOne(this, LaunchModeActivity.SINGLETOP);
    }

}

File: app/src/main/java/io/tanjundang/study/knowledge/launchmode/SingleTopActivity.kt
package io.tanjundang.study.knowledge.launchmode

import android.os.Bundle
import android.view.View

import io.tanjundang.study.R
import io.tanjundang.study.base.BaseActivity

class SingleTopActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun initView() {
        setContentView(R.layout.activity_single_top)
    }

    override fun initData() {

    }

    fun SkipToOne(v: View) {
        OneActivity.SkipToActivityOne(this, LaunchModeActivity.SINGLETOP)
    }

}
